library/core/src/ptr/non_null.rs

```diff
@@ -2300,4 +2318,71 @@ mod verify {
             let distance = ptr_nonnull.offset_from(origin_nonnull);
         }
     }
+
+    #[kani::proof_for_contract(NonNull::byte_add)]
+    pub fn non_null_byte_add_proof() {
+        const ARR_SIZE: usize = mem::size_of::<i32>() * 1000;
```

Comment:
Could you please add a comment explaining why you deem "1000" to be the right number?

Reply:
> Could you please add a comment explaining why you deem "1000" to be the right number?

Thanks for the suggestion @tautschnig, I have updated the code in latest commit